### ISSUE TITLE
New quiz feature support.

### ISSUE THREAD

**Issue by lucasmwadley** [ksucpea/bettercanvas#52]
Canvas updated some of the quizzes in a recent update, Better Canvas does not respond to them or update to dark mode in surrounding areas.

**Comment by ksucpea** [comment]
Not sure how to get to one of these new quizzes because mine are still all the old version. Do you know how to get the css selectors for the elements?

**Comment by lucasmwadley** [comment]
element {
  border: 0px;
  width: 100%;
  height: 1420px;
  display: block;
}

If this isn't what you mean feel free to let me know. Another issue is I'm blocked from viewing the quiz since I submitted it. I can try to let a prof know to make a "test" one so I can look further into it while it's active.

**Comment by ksucpea** [comment]
Would you be willing to use the firefox version and do the dark mode inspection (in the Report an issue tab)? I just added this tool and firefox allows for instant updates unlike chrome.

**Comment by lucasmwadley** [comment]
Firefox, just didn't refresh my updates. great feature. class=.,id=#instructure_ajax_error_box{background: background-0, color: text-0}

**Comment by ksucpea** [comment]
Did you do the inspector on the quiz page and only press it only once? The output seems to be irrelevant, I would've expected it to be a lot. Also the gaps should change color when you press it
